<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemsTab</class>
 <widget class="QWidget" name="SystemsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>239</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="vboxMain">
   <item>
    <widget class="QGroupBox" name="grpDMG">
     <property name="title">
      <string>&amp;Game Boy Title Screens</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="lblDMG">
        <property name="text">
         <string>Game &amp;Boy:</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="buddy">
         <cstring>cboDMG</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cboDMG">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Game Boy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Game Boy Color</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblSGB">
        <property name="text">
         <string>&amp;Super Game Boy:</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="buddy">
         <cstring>cboSGB</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cboSGB">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>Game Boy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Super Game Boy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Game Boy Color</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblCGB">
        <property name="text">
         <string>Game Boy &amp;Color:</string>
        </property>
        <property name="buddy">
         <cstring>cboCGB</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cboCGB">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>Game Boy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Super Game Boy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Game Boy Color</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="lblDMGDescription">
        <property name="text">
         <string>Select the Game Boy model to use for title screens for different types of Game Boy ROM images.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="vspcDownloads">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>197</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cboDMG</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SystemsTab</receiver>
   <slot>comboBox_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cboSGB</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SystemsTab</receiver>
   <slot>comboBox_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cboCGB</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SystemsTab</receiver>
   <slot>comboBox_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>modified()</signal>
  <slot>reset()</slot>
  <slot>loadDefaults()</slot>
  <slot>save()</slot>
  <slot>comboBox_changed()</slot>
 </slots>
</ui>
